tests/unit/App.spec.ts

```diff
@@ -24,6 +24,7 @@ import mockRouter from './MockRouter'
 import { createPinia, setActivePinia } from 'pinia'
 import { useStore } from '@/store/store'
 import { CorpTypeCd, FilingTypes } from '@/enums'
+import { vi } from 'vitest'
```

Comment:
I think there's a way to include vi without needing to explicitly include it in the test? don't know how though

Reply:
I think we can achieve that by using `vitest.xxx` and in this case, no need to import vi. Shall I make this change Travis?

Reply:
I let this one past when you did Create UI -- I don't care, either way.